hs_core/templates/pages/create-resource.html

```diff
@@ -194,13 +194,13 @@ <h2 id="file-type-error" style="color:red">{{ file_validation_error }}</h2>
                 ext = fileList[i].name.match(/\.([^\.]+)$/)[1];
                 ext = "." + ext;
                 var ext_found = false;
-                if (ext === file_types) {
+                if (ext.toLowerCase() === file_types) {
                     ext_found = true;
                 }
                 else {
                     var index;
                     for (index = 0; index < file_types.length; index++) {
-                        if (ext === file_types[index].trim()) {
+                        if (ext.toLowerCase() === file_types[index].trim()) {

```

Comment:
Since this toLowerCase() call is needed in two places, it'd be better to do it once prior to the if-else condition. You can change the statement on line 195 to be "ext = "." + ext.toLowerCase() to make sure ext is already converted to lower case before the if-else check. The same comment applies to genericresource.html change as well.